src/languages/en.js

```diff
@@ -8,7 +8,7 @@ export default {
         attachment: 'Attachment',
         to: 'To',
         optional: 'Optional',
-        split: 'Split',
+        split: ({amount}) => `Split ${amount}`,
```

Comment:
I think it shouldn't be common anymore, as it must have an amount. It should be listed beneath `iou.request` IMO

Reply:
The fact that it has an amount does not make a difference. We are using this in 3 places, so it is pretty common to me.

Reply:
I see `iou.request` used in the same places as `common.split` - both in `IOUConfirmationList.js` and `IOUModal.js`.

My opinion is that adding an `${amount}` in the text makes it very `iou`-related so I'd agree with Jules.